plugins/org.jkiss.dbeaver.ui/META-INF/MANIFEST.MF

```diff
@@ -37,6 +37,7 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.e4.ui.css.swt.theme,
  org.eclipse.e4.ui.workbench.renderers.swt,
  org.jkiss.dbeaver.model,
- org.eclipse.ui.forms
+ org.eclipse.ui.forms,
+ org.eclipse.ui.workbench.texteditor
```

Comment:
Why is this bundle required?

Reply:
We need it to use constants to check search type such as SEARCH_NEXT, SEARCH_PREVIOUS

Reply:
Can it be added to the `org.jkiss.dbeaver.erd.ui` instead?

Reply:
Constants from `IWorkbenchActionDefinitionIds` are used in `ProgressPageControl` which comes from `org.jkiss.dbeaver.ui.controls` package. So, no, it can't.